### DISCUSSION TITLE
问一下

### DISCUSSION THREAD

**Discussion by qazxcgy** [MetaCubeX/ClashMetaForAndroid#202] [Q&A]
![S40406-071256](https://github.com/MetaCubeX/ClashMetaForAndroid/assets/139515302/ea8475b5-8247-4393-b100-3446b324ce1e)
问一下，我的clash机场订阅显示是
Get "https://dlink.host/1drv/aHR0cHM6Ly8xZHJ2Lm1zL3UvcyFBbm5oU0ZGT0NGX05pQ2ljR0d6RGNYQWRWQUkwP2U9U1RzZEVR.jpg": tls: failed to verify certificate:x509: certificate has expired oris not yet valid: current time2025-03-04T10:04:10+-8000:00 isafter 2021-09-30T14:01:15z
哪位大佬知道什么意思嘛

**Comment by TSaltedfishKing** [comment]
Website's certificate outdated. Please ask the service provider.

Further tips: Before you put forward this question, you should make a attempt to search the error messages on internet. If there's answer on internet, it must be faster than ask here.

Also, I am happy to provide help for you.

**Reply by qazxcgy** [reply]
你好，你是中国人吗，
如何在互联网上搜索答案，我不知道他是什么样的错误，

**Reply by TSaltedfishKing** [reply]
[xiaoyangdkj](https://github.com/xiaoyangdkj) had put the right answer here.

**Comment by xiaoyangdkj** [comment]
> 
> ![S40406-071256](https://github.com/MetaCubeX/ClashMetaForAndroid/assets/139515302/ea8475b5-8247-4393-b100-3446b324ce1e)
> 问一下，我的clash机场订阅显示是
> Get "https://dlink.host/1drv/aHR0cHM6Ly8xZHJ2Lm1zL3UvcyFBbm5oU0ZGT0NGX05pQ2ljR0d6RGNYQWRWQUkwP2U9U1RzZEVR.jpg": tls: failed to verify certificate:x509: certificate has expired oris not yet valid: current time2025-03-04T10:04:10+-8000:00 isafter 2021-09-30T14:01:15z
> 哪位大佬知道什么意思嘛

手机系统版本有点老了（内置证书过期），可以浏览器下载配置文件后手动导入
